app/database/decoder/breeze_buddy/template.py

```
@@ -3,14 +3,130 @@
+    """Convert old-format voice fields to the new unified tts_configuration.
+
+    Handles templates stored with the old schema:
+      - tts_voice_name: "rhea" / "mira" / "sara"
+      - mira_voice_id: str
+      - cartesia_voice_configurations: {...}
+      - elevenlabs_voice_configurations: {...}
+
+    Transforms them into tts_configuration + tts_configuration_overrides.
+    Already-migrated templates (with tts_configuration) pass through unchanged.
+    """
+    if "tts_configuration" in data and data["tts_configuration"]:
```

Ruff (0.15.9)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)